src/views/System/Actionlog/Index.vue

```diff
@@ -153,7 +157,7 @@
               <b-card :header="$t('details.header')">
                 <b-row>
                   <b-col cols="4">
-                    ID
+                    {{ $t('details.id') }}
                   </b-col>
                   <b-col cols="8">
                     {{ a.actionID }}
```

Comment:
In the same file, on line 288, there is a `severity` object. 
The labels on there need to be translated too. 
So for example, `emergency` -> `this.$t('severity.emergency')`